src/components/popover/popover.component.scss

```diff
@@ -3,6 +3,8 @@
     flex-direction: column;
     max-width: none;
     padding: 0;
+    left: calc(50% - 285px);
```

Comment:
Where is this 285 px come from? Please add comment or constant to the code the explain the value.